verl/workers/rollout/vllm_rollout/vllm_rollout.py

```diff
@@ -229,11 +229,11 @@ def generate_sequences(self, prompts: DataProto, **kwargs) -> DataProto:
             # TODO(sgm): disable logprob when recompute_log_prob is enable
             # if n = 1: (bs, response_length) ; if n > 1: (bs * n, response_length)
             response = output[0].to(idx.device)
-            # log_probs = output[1].to(idx.device)
+            log_probs = output[1].to(idx.device)
```

Comment:
Do we need to check if logprob is >0 here?

Reply:
logprob will never be > 0?